lib/takeaway.rb

```diff
@@ -0,0 +1,51 @@
+require_relative 'current_order'
+require 'twilio-ruby'
+
+class Takeaway
+
+  attr_reader :menu
+
+  def initialize
+    @menu = [{ item: 'Mega Breakfast', price: 6.75 },
+              { item: 'Full English Breakfast', price: 6.65 },
+              { item: 'Light Breakfast', price: 4.55 },
+              { item: 'Veggie Breakfast', price: 5.95 },
+              { item: 'Breakfast Special', price: 6.85 },
+              { item: 'Cheeseburger', price: 7.25 },
+              { item: 'BBQ Burger', price: 7.85 },
+              { item: 'Mega Burger', price: 8.75 },
+              { item: 'Vegatarian Burger', price: 5.95 }]
+    @current_order = CurrentOrder.new
+  end
+
+  def choose_meal(meal)
+    @current_order.add_item(@menu.find { |n| n[:item] == meal })
+  end
+
+  def confirm_order
+    @current_order.total
+  end
+
+  def pay_for_order
+    send_message
+    @current_order.clear_order
+  end
+
+ private
+
```

Comment:
Good job implementing text messaging!

Reply:
Did you consider creating a separate class for this?  Maybe a `Notification` class?